src/v/cloud_storage/tests/remote_test.cc

```diff
@@ -145,87 +141,106 @@ class remote_fixture_base
+class all_types_remote_fixture
+  : public remote_fixture
+  , public testing::TestWithParam<remote_test_parameters> {
+public:
+    all_types_remote_fixture()
+      : remote_fixture(GetParam().url_style) {}
+};
+
+class all_types_gcs_remote_fixture
+  : public gcs_remote_fixture
+  , public testing::TestWithParam<remote_test_parameters> {
+public:
+    all_types_gcs_remote_fixture()
+      : gcs_remote_fixture(GetParam().url_style) {}
+};
+
+TEST_P(all_types_remote_fixture, test_download_manifest_json) {
     set_expectations_and_listen({expectation{
-      .url = "/" + manifest_url, .body = ss::sstring(manifest_payload)}});
+      .url = manifest_url, .body = ss::sstring(manifest_payload)}});
     auto subscription = remote.local().subscribe(allow_all);
     run_manifest_download_and_check(
       remote,
+      bucket_name,
       load_manifest_from_str(manifest_payload),
       manifest_format::json,
       "manifest load from json");
-    BOOST_CHECK(subscription.available());
-    BOOST_CHECK(
+    EXPECT_TRUE(subscription.available());
+    EXPECT_TRUE(
       subscription.get().type == api_activity_type::manifest_download);
 }
 
-FIXTURE_TEST(test_download_manifest_serde, remote_fixture) {
+TEST_P(all_types_remote_fixture, test_download_manifest_serde) {
     auto translator = load_manifest_from_str(manifest_payload);
     auto serialized = translator.serialize().get();
     auto manifest_binary
       = serialized.stream.read_exactly(serialized.size_bytes).get();
 
     set_expectations_and_listen({expectation{
-      .url = "/" + manifest_serde_url,
+      .url = manifest_serde_url,
       .body = ss::sstring{manifest_binary.begin(), manifest_binary.end()}}});
 
     auto subscription = remote.local().subscribe(allow_all);
     run_manifest_download_and_check(
       remote,
+      bucket_name,
       std::move(translator),
       manifest_format::serde,
       "manifest load from serde");
 
-    BOOST_CHECK(subscription.available());
-    BOOST_CHECK(
+    EXPECT_TRUE(subscription.available());
+    EXPECT_TRUE(
       subscription.get().type == api_activity_type::manifest_download);
 }
 
-FIXTURE_TEST(test_download_manifest_timeout, remote_fixture) { // NOLINT
+TEST_P(all_types_remote_fixture, test_download_manifest_timeout) { // NOLINT
     partition_manifest actual(manifest_ntp, manifest_revision);
     auto subscription = remote.local().subscribe(allow_all);
     retry_chain_node fib(never_abort, 100ms, 20ms);
     auto res = remote.local()
                  .download_manifest(
-                   cloud_storage_clients::bucket_name("bucket"),
-                   json_manifest_format_path,
-                   actual,
-                   fib)
+                   bucket_name, json_manifest_format_path, actual, fib)
                  .get();
-    BOOST_REQUIRE(res == download_result::timedout);
-    BOOST_REQUIRE(subscription.available());
-    BOOST_REQUIRE(
+    ASSERT_TRUE(res == download_result::timedout);
```

Comment:
same question here, the test is a fixture test which is based on Boost.Test but this looks like a macro from GTest, is it OK to mix them?

Reply:
looks like this is a gtest now

Reply:
Yes, this was all migrated to GTest for ease of parameterization.